core/pkg/server/sender.go

```diff
@@ -1063,6 +1064,18 @@ func (s *Sender) sendFile(file *service.FilesItem) {
 	}
 }
 
+func (s *Sender) sendArtifact(record *service.Record, msg *service.ArtifactRecord) {
+	saver := artifacts.NewArtifactSaver(
+		s.ctx, s.graphqlClient, s.fileTransferManager, msg, 0, "",
+	)
+	artifactID, err := saver.Save(s.fwdChan)
+	if err != nil {
+		err = fmt.Errorf("sender: sendArtifact: failed to log artifact ID: %s; error: %s", artifactID, err)
+		s.logger.CaptureError("sender received error", err)
```

Comment:
this will send a sentry event, do you want to do it? or just track it in our logs?

Reply:
oh lets just track it in our logs. how do i do that? do i just remove the `CaptureError` call?